comparisons/validateJR.sh

```diff
@@ -21,20 +21,23 @@ ls -d ${baseA}/[1-9]* | while read -r d; do
   echo "${dL}" | grep  " $d/" >& /dev/null || echo $d >> missing_map.txt
 done
 
+mkdir -p validate_lib
+pushd validate_lib
+    cp ${VALIDATE_C_SCRIPT} ./validate.C
+    echo -e "gSystem->Load(\"libFWCoreFWLite.so\");\n AutoLibraryLoader::enable();\n FWLiteEnabler::enable();\n .L validate.C+\n .qqqqqq" | root -l -b
+popd
 grep root ${inList} | grep -v "#" | while read -r dsN fNP procN comm; do
     #fNP can be a pattern: path-expand it first
     fN=`echo ${baseA}/${fNP} | cut -d" " -f1 | sed -e "s?^${baseA}/??g"`
     #[ ! -f "${baseA}/${fN}" ] && echo Missing ${baseA}/${fN}
     #process regular files first; need files both in baseA and baseB
     if [ -f "${baseA}/${fN}" -a -f "${baseB}/${fN}" ]; then
-	extN=all_${diffN}_${dsN}
-	mkdir -p ${extN}
-	pushd ${extN}
-	  cp ${VALIDATE_C_SCRIPT} ./
-	  echo "Will run on ${fN} in ${extN}"
-	  g++ -shared -o validate.so validate.C `root-config --cflags ` -fPIC
```

Comment:
just for posterity, this part was removed in local test setups in 2015 https://github.com/slava77/cms-reco-tools/commit/8129c94139793bd111359e64badeb857ef364c1e
I guess it never fully propagated to this repo